seeker/host/fstab.go

```diff
@@ -0,0 +1,38 @@
+package host
+
+import (
+	"fmt"
+
+	"github.com/hashicorp/hcdiag/seeker"
+)
+
+var _ seeker.Runner = Fstab{}
+
+type Fstab struct {
+	os      string
+	sheller *seeker.Seeker
```

Comment:
@mkcp and I had an offline conversation about this, but I'm capturing it here for posterity. The real reason I included these bits of state was so that it would be easier to test the behavior of the `FSTab` struct. Without doing something like this, I didn't know of a good way to mock out the underlying sheller that these seeker relies on.